Publish the plugin for Java8

When using the plugin in a project that uses Java8, Maven complains with the following error:

[ERROR] Failed to execute goal com.github.funthomas424242:plantuml-maven-plugin:1.5.1:generate (render-plantuml) on project app: Execution render-plantuml of goal com.github.funthomas424242:plantuml-maven-plugin:1.5.1:generate failed: Unable to load the mojo 'generate' in the plugin 'com.github.funthomas424242:plantuml-maven-plugin:1.5.1' due to an API incompatibility: org.codehaus.plexus.component.repository.exception.ComponentLookupException: com/github/plantuml/maven/PlantUMLMojo has been compiled by a more recent version of the Java Runtime (class file version 55.0), this version of the Java Runtime only recognizes class file versions up to 52.0

The reason is that the plugin was compiled and targeted for Java 11.
Setting compiler target to 1.8 should fix  this

--- a/pom.xml
+++ b/pom.xml
@@ -5,7 +5,7 @@
 
     <groupId>com.github.funthomas424242</groupId>
     <artifactId>plantuml-maven-plugin</artifactId>
-    <version>1.5.1</version>
+    <version>1.5.2-SNAPSHOT</version>
     <packaging>maven-plugin</packaging>
 
     <name>Maven PlantUML plugin</name>
@@ -55,7 +55,6 @@
         <mavenVersion>3.6.3</mavenVersion>
         <java.build.version>13</java.build.version>
         <java.target.version>11</java.target.version>
-        <maven.compiler.release>${java.target.version}</maven.compiler.release>
     </properties>
 
     <dependencies>
@@ -331,6 +330,33 @@
     </build>
 
     <profiles>
+        <profile>
+            <id>java-8</id>
+            <activation>
+                <jdk>1.8</jdk>
+            </activation>
+            <properties>
+                <maven.compiler.target>1.8</maven.compiler.target>
+                <java.build.version>1.8</java.build.version>
+                <java.target.version>1.8</java.target.version>
+                <maven.compiler.source>1.8</maven.compiler.source>
+            </properties>
+            <build>
+                <plugins>
+                    <plugin>
+                        <groupId>org.apache.maven.plugins</groupId>
+                        <artifactId>maven-compiler-plugin</artifactId>
+                        <configuration combine.self="override">
+                            <encoding>UTF-8</encoding>
+                            <compilerArgs>
+                                <arg>-Xlint:all</arg>
+                            </compilerArgs>
+                            <fork>true</fork>
+                        </configuration>
+                    </plugin>
+                </plugins>
+            </build>
+        </profile>
         <profile>
             <id>release</id>
             <build>

